driver/razerkbd_driver.h

```diff
@@ -80,6 +80,7 @@
 #define USB_DEVICE_ID_RAZER_CYNOSA_V2 0x025E
 #define USB_DEVICE_ID_RAZER_HUNTSMAN_V2_ANALOG 0x0266
 #define USB_DEVICE_ID_RAZER_BOOK_2020 0x026A
+#define USB_DEVICE_ID_RAZER_BLADE_15_BASE_EARLY_2021 0x026F
```

Comment:
Move this one down to keep it sorted by value

Reply:
Changes made based on feedback.